tests/responses/pages-responses.js

```diff
@@ -63,4 +77,40 @@ export default {
       },
     },
   },
+
+  boron2: {
+    data: {
```

Comment:
I urge you to simplify mock test data. IMHO, good test data shows a single example of important bits, and no more.

Reply:
OK I will work on that.

Reply:
Also, I would prefer names and values that highlight why the data item is in there, rather than real-world examples.